File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/composite/transport/Bike.java
package com.bigboxcode.designpattern.composite.transport;

public class Bike implements Transport{
    @Override
    public void start() {
        System.out.println("Starting Bike...");
    }

    @Override
    public void operate() {
        System.out.println("Riding Bike");
    }

    @Override
    public void stop() {
        System.out.println("Stopping Bike...");
    }
}

File: design-patterns/php/composite/transportlist/Bike.php
<?php
// Bike.php

namespace BigBoxCode\DesignPattern\Composite\TransportList;


class Bike implements Transport {
    public function start(): void {
        echo "Starting Bike...\n";
    }

    public function operate(): void {
        echo "Riding Bike\n";
    }

    public function stop(): void {
        echo "Stopping Bike...\n";
    }
}